File: exercicio_1010.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        // Leitura do primeiro item
        int cod1 = sc.nextInt();
        int qtd1 = sc.nextInt();
        double valor1 = sc.nextDouble();

        // Leitura do segundo item
        int cod2 = sc.nextInt();
        int qtd2 = sc.nextInt();
        double valor2 = sc.nextDouble();

        double total = (qtd1 * valor1) + (qtd2 * valor2);

        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
    }
}

File: exercicio_1010.c
#include <stdio.h>
int main() {
 int cod1, cod2, qtd1, qtd2; 
 double valor1, valor2, total;
 
//Entrada de dados:
 scanf("%d %d %lf", &cod1, &qtd1, &valor1);
 
 scanf("%d %d %lf", &cod2, &qtd2, &valor2);

//Calculo total: 
 total =(qtd1*valor1)+(qtd2*valor2);

//Saida de dados: 
 printf("VALOR A PAGAR : R$ %.2lf\n",total);

    return 0;
}
